framework/tests/all-systems/java8inference/Issue2975.java

```diff
@@ -0,0 +1,22 @@
+package sup;
```

Comment:
Is the package here necessary? It doesn't seem to match the location of the file.